web-api/src/storage/elastic.rs

```diff
@@ -356,10 +362,70 @@ impl Client {
         }));
 
         Ok(self
-            .query_with_json::<_, IgnoredResponse>(url, body)
+            .query_with_json::<_, IgnoredResponse>(Method::POST, url, body)
             .await?
             .map(|_| ()))
     }
+
+    pub(super) async fn extend_elasticsearch_mapping(
```

Comment:
:art: we could skip the name as it is already a function on the es client.
```suggestion
    pub(super) async fn extend_mapping(
```